packages/xo-web/src/xo-app/xoa/support/index.js

```diff
@@ -22,35 +26,83 @@ const Support = decorate([
       }),
       checkXoa: async () => ({ stdoutCheckXoa: await checkXoa() }),
     },
+    computed: {
+      stdoutSupportTunnel: (_, { tunnelState }) =>
+        tunnelState === undefined
+          ? undefined
+          : { __html: marked(tunnelState.stdout) },
```

Comment:
Why is it `marked` and not `ansi_to_html`ed?

Reply:
for homogeneity, I changed `marked` to `ansi_to_html`.